feat(changelog): containers-added-serverless-containers-new-autoscaling- 2025-02-26 (#4502)
scaleway/docs-content: changelog/february2025/2025-02-26-containers-added-serverless-containers-new-autoscaling-.mdx

---

> title: Serverless Containers new Autoscaling Rules

Reviewer: ```suggestion
title: Serverless Containers new autoscaling rules
```

status: added
date: 2025-02-26
category: serverless
product: containers
---

> 
> [Autoscaling](https://www.scaleway.com/en/docs/serverless-containers/concepts/#autoscaling) now supports new rules: CPU and RAM percentage. 
> 
> This feature allows users to automatically adjust the CPU and RAM resources allocated to their Serverless Containers based on workload demands. 
> 
> 
> 

Reviewer: ```suggestion

[Autoscaling](https://www.scaleway.com/en/docs/serverless-containers/concepts/#autoscaling) now supports new rules: CPU and RAM percentage. 

This feature allows you to automatically adjust the CPU and RAM resources allocated to your Serverless Containers based on workload demands. 

```
